src/qt/trafficgraphwidget.cpp

```diff
@@ -65,6 +65,7 @@ void TrafficGraphWidget::paintPath(QPainterPath &path, QQueue<float> &samples)
 void TrafficGraphWidget::paintEvent(QPaintEvent *)
 {
     QPainter painter(this);
+    painter.setRenderHints(painter.renderHints() | QPainter::Antialiasing);
```

Comment:
I believe configuration should come before usage.

Reply:
You can change render hints at any time.

Anyway, you can simplify this:
```cpp
painter.setRenderHint(QPainter::Antialiasing);
```

Reply:
You can, but should you?

Your suggestion is good, pushing.

Reply:
> You can, but should you?

It depends if `fillRect` is faster without antialiasing - in this case it might make no difference as the rect is aligned and fills the entire surface.